app/experimenter/legacy-ui/rapid/components/visualization/HighlightsTable.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+
+import { featureOptions } from "experimenter-rapid/components/forms/ExperimentFormOptions";
+import {
+  METRIC,
+  HIGHLIGHTS_METRICS_LIST,
+  BRANCH_COMPARISON,
+  TABLE_LABEL,
+} from "experimenter-rapid/components/visualization/constants/analysis";
+import {
+  METRICS_TIPS,
+  SEGMENT_TIPS,
+} from "experimenter-rapid/components/visualization/constants/tooltips";
+import ResultsRow from "experimenter-rapid/components/visualization/ResultsRow";
+import { ExperimentData } from "experimenter-types/experiment";
+
+const getHighlightMetrics = (
+  featureData: Array<string>,
+): Array<{ value: string; name: string; tooltip: string }> => {
```

Comment:
You may very well know this, but just in case, and if you find it handy: there's a shorthand array syntax! `{ value: string; name: string; tooltip: string }[]` (totally fine if you prefer the `Array<>` way tho)